packages/frontend/app/about-us/components/about-us.tsx

```diff
@@ -0,0 +1,119 @@
+import { Bug, Mails, Phone } from "lucide-react";
+import Image from "next/image";
+
+import { ArticleSection } from "./article-section";
+import { TeamMemberCard } from "./team-member-card";
+
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+
+export function AboutUs() {
+  return (
+    <article className="w-full text-center">
+      <header className="relative w-full h-[200px]">
+        <Image
+          src="/about-us-header.jpg"
+          alt="idek"
+          layout="fill"
+          objectFit="cover"
+          className="brightness-50"
+        />
+        <h1 className="w-full absolute top-1/2 bottom-1/2 m-auto font-bold text-4xl md:text-6xl select-none">
+          About
+        </h1>
+      </header>
+      <main>
+        <section>
+          <h2 className="font-bold text-4xl mt-4">Who we are</h2>
+          <div className="w-full flex justify-center gap-x-8 mt-4">
+            <TeamMemberCard
+              teamMember="Jude Davis"
+              memberDesc="
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed tempus mauris a sapien commodo, quis rutrum magna tincidunt. Suspendisse tempus ut orci a efficitur. Proin vitae ante at ex sagittis gravida. Nullam semper nisl vel blandit fringilla."
+            />
+            <TeamMemberCard
+              teamMember="Jonathan Koziel"
+              memberDesc="
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed tempus mauris a sapien commodo, quis rutrum magna tincidunt. Suspendisse tempus ut orci a efficitur. Proin vitae ante at ex sagittis gravida. Nullam semper nisl vel blandit fringilla."
+            />
```

Comment:
As discussed perhaps not our names here, but a link to the org and features/products as we create them here.